x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/40_engine_delete.yml

```diff
@@ -0,0 +1,30 @@
+setup:
+  - do:
+      indices.create:
+        index: test-index
+        body:
+          settings:
+            index:
+              number_of_shards: 1
+              number_of_replicas: 0
+  - do:
+      engine.put:
+        engine_id: test-engine-to-delete
+        body:
+          indices: ["test-index"]
+
+---
+"Delete Engine":
+  - do:
+      engine.delete:
+        engine_id: test-engine-to-delete
+
+  - match: { acknowledged: true }
```

Comment:
Would it make sense to do a GET afterwards and catch a "missing" exception to ensure it was deleted?